grafana-node-exporter-dashboard.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Node Exporter 面板重要监控指标详解
+aliases: ['/docs-cn/dev/grafana-node-exporter-dashboard/','/docs-cn/dev/reference/key-monitoring-metrics/node-exporter-dashboard/']
+---
+
+# Node Exporter 面板重要监控指标详解
+
+使用 TiUP 部署 TiDB 集群时，一键部署监控系统 (Prometheus & Grafana)，监控架构参见 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
+
+目前 Grafana Dashboard 整体分为 PD、TiDB、TiKV、Node_exporter、Overview 等。
+
+以下为 Node Exporter Dashboard 监控说明：
+
+## Overview
+
+- Virtual CPUs: CPU 核心数量
+- Total RAM: 内存总大小
+- Total Swap: 可使用的交换内存大小
+- System Uptime: 系统启动时长
+- Node_exporter version: Node Exporter 使用版本
+- CPU Used: CPU 的使用率
+- Memory Used: 内存的使用率
+- Swap Used: 交换内存的使用率
+- Load1: 1 分钟的负载情况
+- Load5: 5 分钟的负载情况
+- Load15: 15 分钟的负载情况
+
+## Kernel
+
+- OS: 操作系统类型
+- Machine: 机器平台
+- Kernel version: 内核版本
+- Interrupts: 内核中断情况
+- Forks: Linux 下创建进程的系统调用情况
+- Context Switches: 内核任务切换情况
```

Comment:
A context switch (also sometimes referred to as a process switch or a task switch) is the switching of the CPU (central processing unit) from one process or thread to another.